### PR TITLE
feat(provider/oraclebmcs): Add instance details to server groups

### PR DESCRIPTION
**Pull request by owainlewis**
Adds basic instance details information to server groups view.

screen shot 2017-05-17 at 16 44 09

### CODE REVIEW

File: app/scripts/modules/oracle/instance/details/instance.details.controller.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+import {INSTANCE_READ_SERVICE} from 'core/instance/instance.read.service';
+import {INSTANCE_WRITE_SERVICE} from 'core/instance/instance.write.service';
+import {CLOUD_PROVIDER_REGISTRY} from 'core/cloudProvider/cloudProvider.registry';
+
+let angular = require('angular');
```
Comment (anotherchrisberry) [inline app/scripts/modules/oracle/instance/details/instance.details.controller.js]: just pushed a rather large PR detailing new import structure - can you roll these into:
```javascript
const angular = require('angular');

import { 
  CLOUD_PROVIDER_REGISTRY, 
  INSTANCE_READ_SERVICE, 
  INSTANCE_WRITE_SERVICE 
} from '@spinnaker.core';
```

Comment (owainlewis) [inline app/scripts/modules/oracle/instance/details/instance.details.controller.js]: @anotherchrisberry sure that change looks great! 👍  I'll wait until your changes are merged and then update this.

Comment (owainlewis) [inline app/scripts/modules/oracle/instance/details/instance.details.controller.js]: @anotherchrisberry Fixed. Great change 👍

**Comment by anotherchrisberry** [comment]
LGTM - thanks - I've got one more PR to get in (https://github.com/spinnaker/deck/pull/3697); after that, rebase this and we'll get it merged before we push anything else.

**Comment by owainlewis** [comment]
@anotherchrisberry rebased

**Pull request merged**